requirements/constraints.txt

```diff
@@ -24,8 +24,8 @@ django-celery-results<2.1
 # We do not support version django-config-models<1.0.0
 django-config-models>=1.0.0
 
-# The CORS_ORIGIN_WHITELIST changes in a backwards incompatible way in 3.0.0, needs matching configuration repo changes
-django-cors-headers<3.0.0
+# greater version has breaking changes. check configuration changes as well when upgrade.
```

Comment:
When you say `check configuration changes` what do you mean?  Is there a specific thing you can link here that will make it easier for people to see what you mean? Maybe the PR that updates the settings files?

Reply:
For existing PR I have merged these changes. For greater there are more changes, I will add the more details there.
```
https://github.com/edx/edx-internal/pull/5369
https://github.com/edx/sandbox-internal/pull/189
```

Reply:
I have updated the comment. v`3.2.0` is fine for us to run `django30` tests. 

3.5.0 has major change again.
```
Following Django’s example in Ticket #31670 for replacing the term “whitelist”, 
plus an aim to make the setting names more comprehensible, the following settings have been renamed:

CORS_ORIGIN_WHITELIST -> CORS_ALLOWED_ORIGINS
CORS_ORIGIN_REGEX_WHITELIST -> CORS_ALLOWED_ORIGIN_REGEXES
CORS_ORIGIN_ALLOW_ALL -> CORS_ALLOW_ALL_ORIGINS
```
https://github.com/adamchainz/django-cors-headers/blob/main/HISTORY.rst#350-2020-08-25